ethpm_types/manifest.py

```diff
@@ -263,3 +263,37 @@ def unpack_sources(self, destination: Path):
             source_path.parent.mkdir(parents=True, exist_ok=True)
 
             source_path.write_text(content)
+
+    def get_contract_compiler(self, contract_type_name: str) -> Optional[Compiler]:
+        """
+        Get the compiler used to compile the contract type, if it exists.
+
+        Args:
+            contract_type_name (str): The name of the compiled contract.
+
+        Returns:
+            Optional[`~ethpm_types.source.Compiler`]
+        """
+        for compiler in self.compilers or []:
+            if contract_type_name in (compiler.contractTypes or []):
+                return compiler
+
+        return None
+
+    def update_compilers(self, compilers: List[Compiler]):
```

Comment:
could also be `extend_compilers` ?

Reply:
maybe just `add_compilers`

Reply:
and could be `add_compilers(*compilers: Sequence[Compiler])`